script.js

```diff
@@ -1,12 +1,82 @@
-//You can edit ALL of the code here
-function setup() {
-  const allEpisodes = getAllEpisodes();
-  makePageForEpisodes(allEpisodes);
-}
-
-function makePageForEpisodes(episodeList) {
-  const rootElem = document.getElementById("root");
-  rootElem.textContent = `Got ${episodeList.length} episode(s)`;
-}
-
-window.onload = setup;
+//You can edit ALL of the code here
+
+function setup() {
+  const selection = document.getElementById("bar");
+  selection.addEventListener("change", () => {
+    showTech(selection.value);
+  });
+  const img = document.getElementById("logo");
+  img.addEventListener("click", () => {
+    window.location = "./";
+  });
+}
+
+window.onload = setup;
+
+function clearContent() {
+  document.querySelectorAll("#container").forEach((el) => el.remove());
+}
+function selectionChange(selValue) {
+  const selection = document.getElementById("bar");
+  selection.value = selValue;
+  selection.dispatchEvent(new Event("change"));
+}
+function showTech(selValue) {
+  clearContent();
+
+  const mainEl = document.getElementById("mainContent");
+
+  const containEl = document.createElement("div");
+  containEl.id = "container";
+
+  const divEl1 = document.createElement("p");
+  divEl1.id = "hero-text";
+  divEl1.classList.add("hero-text-middle");
+  divEl1.innerText = headers[selValue];
+  containEl.appendChild(divEl1);
+
+  const sectionEl = document.createElement("section");
+  sectionEl.className = "section";
+
+  let dataArr = [];
+  if (selValue == "techOpt") dataArr = getAllTechs();
+  else if (selValue == "aniOpt") dataArr = getAllAnimals();
+  else if (selValue == "recOpt") dataArr = getAllRecycles();
+  else {
+    window.location = "./";
+  }
+
+  dataArr.forEach((obj) => {

```

Comment:
good looping through of the array to create cards. You could even extract the logic out of this function (maybe using a function called `createCard(obj)` for example) and call this function within the forEach loop.